### ISSUE TITLE
When I try to set `attributes` in `authlete_client`, I can plan but apply fails.

### ISSUE THREAD

**Issue by intercept6** [authlete/terraform-provider-authlete#111]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
1.4.6

### Affected Resource(s)
- authlete_client

### Terraform Configuration Files
```hcl
resource "authlete_client" "my_client" {
  attributes {
    key   = "my_key"
    value = "my_value"
  }
}
```

### Panic Output
https://gist.github.com/intercept6/57d17cf0caa255a5b7142b3422bcd2c5

### Expected Behavior
Attributes can be set.

### Actual Behavior
Can't do it.

### Steps to Reproduce
1. `terraform apply`

### Important Factoids
Running on Terraform Cloud.
It uses Authlete Enterprise.

### References
The same function is used to generate the schema for service attributes and client attributes, but they are written in different ways to handle changes.

service attribute
https://github.com/authlete/terraform-provider-authlete/blob/4ca28936a1f38a2e7d002cc8a95cdfde7c8dbee9/internal/provider/service.go#L31
https://github.com/authlete/terraform-provider-authlete/blob/4ca28936a1f38a2e7d002cc8a95cdfde7c8dbee9/internal/provider/service.go#L252-L254

client attributes
https://github.com/authlete/terraform-provider-authlete/blob/4ca28936a1f38a2e7d002cc8a95cdfde7c8dbee9/internal/provider/client.go#L108
https://github.com/authlete/terraform-provider-authlete/blob/4ca28936a1f38a2e7d002cc8a95cdfde7c8dbee9/internal/provider/client.go#L579-L581

**Comment by hefa** [comment]
@intercept6 Thank you for reporting this issue. We will investigate it immediately.

**Comment by hefa** [comment]
This issue is fixed in version v1.3.8, which is available now.
